### ISSUE TITLE
Final validation on the Wizard forms does not seem to work as expected

### ISSUE THREAD

**Issue by matdar** [creativetimofficial/ct-vue-paper-dashboard-pro#66]
The wizard form has the following code which doesn't seem to get executed when the final wizard tab is complete and the Finish button is clicked:

`wizardComplete() {
        swal('Good job!', 'You clicked the finish button!', 'success')
      }
`

I see this behaviour in my local running instance and on the online demo:
https://demos.creative-tim.com/vue-paper-dashboard-pro

I intend to use a wizard form my dashboard so it would help if the validation worked as it looks like it should.

Thanks
Matt

**Comment by wasadigi** [comment]
I also encountered this issue. I changed the code and got it to work:

```
        <wizard-tab :before-change="() => validateStep('step3')">
          <template slot="label">
            Validate
          </template>
          <third-step
            ref="step3"
            @on-validated="wizardComplete"/>
        </wizard-tab>
```

**Comment by cristijora** [comment]
Hey guys @matdar @wasadigi 
This issue has been addressed and fixed in latest 2.2.0 version.
You can now use the `@complete` event on the wizard component to handle the "finish" button click. The demo page has been adjusted as well and you can check how it's done there.
To upgrade, just download the dashboard again from creative tim (downloads section) and follow [migration guide here](https://demos.creative-tim.com/vue-paper-dashboard-pro/documentation/migration-guide.html#from-2-1-0-to-2-2-0)
